source/slides/22_haproxy.rst

```diff
@@ -0,0 +1,99 @@
+.. _22_haproxy:
+
+HAProxy
+=======
+
+Objectives Today
+----------------
+
+1. Install and setup HAProxy
+2. Setup several backends
```

Comment:
For the backends, you could do something as simple as:

```bash
for x in `seq 8000 8004`; do python -m simpleHTTPServer $x 1>&2 2>/dev/null &; done
```

and then have a bunch of local backends running on the server.